web/server/vue-cli/package.json

```diff
@@ -56,10 +57,10 @@
     "chromedriver": "^89.0.0",
     "copy-webpack-plugin": "^6.0.3",
     "core-js": "^3.10.1",
-    "css-loader": "^3.6.0",
     "cross-env": "^7.0.2",
+    "css-loader": "^3.6.0",
     "deepmerge": "^4.2.2",
-    "eslint": "^7.4.0",
+    "eslint": "^7.32.0",
```

Comment:
Why did you increased this dependency version in this PR? I am not against this, but please create a separate PR for this.